core/sail/shacl/src/test/resources/test-cases/class/predicateObjectTarget/shacl.ttl

```diff
@@ -0,0 +1,27 @@
+@base <http://example.com/ns> .
+@prefix ex: <http://example.com/ns#> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix sh: <http://www.w3.org/ns/shacl#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix rdf4j-sh: <http://rdf4j.org/schema/rdf4j-shacl#> .
+
+ex:PersonShape
+	a sh:NodeShape  ;
+
+        rdf4j-sh:compoundTarget [
+	        rdf4j-sh:targetPredicate rdf:type;
+	        rdf4j-sh:targetObject ex:Person
+	];
+
+        rdf4j-sh:compoundTarget [
+	        rdf4j-sh:targetPredicate rdf:type;
+	        rdf4j-sh:targetObject ex:SecondTarget
+	];
+
+	sh:property [
+		sh:path ex:knows ;
+		sh:and ( [sh:class ex:Person]  [sh:class ex:Human])
+	] .
+
```

Comment:
An example of the new predicate+object target support.

You can have as many `rdf4j-sh:compoundTarget`s as you want, but each one can only have a single predicate and single object. For the time being at least.